### ISSUE TITLE
Why did I get a lot of fake Pokemon location in last hour?

### ISSUE THREAD

**Issue by baomx888** [5andr0/PogoLocationFeeder#258]
It keeps showing Mew/Zapdos  in 100 IV and keep the bot not running. Can we block certain feed?

**Comment by Ilses** [comment]
Yes -- you can remove channels from discord_channels.json to ignore them.

**Comment by saahilz** [comment]
how do you do that

**Comment by Ilses** [comment]
@saahilz Any text editor should do.

**Comment by baomx888** [comment]
thanks!

**Comment by Adriskye** [comment]
or just delete dito and legendaries from necrobot if u use it

**Comment by Sylencer90** [comment]
What exactly need to be delete out? Deleted the whole content of discord channels and it throws errors.

**Comment by Ilses** [comment]
@Sylencer90 It's JSON -- you can't just delete the whole thing. But an empty list is fine. See e.g. http://www.w3schools.com/json/json_syntax.asp

**Comment by Sylencer90** [comment]
So give me an template of what to edit in the list, what to delete and what not.

**Comment by lordlolcommon** [comment]
@Sylencer90 Open up discord_channels.json and remove the channel you don't want pogofeeder to sent you.  If you don't want channel 100_iv from necrobot, remove the whole content. for example:

{
"id": "209314800935763977",
        "Server": "Necrobot.io",
        "Name": "100_iv"
},

remove all of that.
